guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -56,6 +56,11 @@ include::topics/upgrading_discovery_satellite.adoc[leveloffset=+3]
 // Configuring Subnets with a Template Capsule
 include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+3]
 
+ifdef::foreman-el,katello,satellite[]
+// Upgrading Foreman/Katello from EL7 to EL8
+include::common/modules/proc_upgrading-el7-to-el8.adoc[leveloffset=+2]
```

Comment:
Only thought I had is that given this is a big deal to upgrade the OS and that it is required to be able to update to the following version of Foreman/katello/Satellite if it should have its own top level section header to ensure it stands out. 

Right now we have:

```
[1. Upgrade Overview]
[2. Upgrading Foreman]
[3. Updating Foreman server and Content Hosts]
```

Wondering if we need:

```
[1. Upgrade Overview]
[2. Upgrading Foreman]
[3. Upgrading Foreman from Enterprise Linux 7 to Enterprise Linux 8]
[4. Updating Foreman server and Content Hosts]
```

Reply:
@bangelic I will review after you address and resolve this comment.